charts/argocd-operator/templates/anyuid-scc.yaml

```diff
@@ -1,29 +1,16 @@
 {{- if .Values.argocd_cr.ha.enabled }}
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
+kind: RoleBinding
 metadata:
-  name: {{ .Values.namespace }}-argocd-redis-ha-anyuid-scc
-rules:
-- apiGroups:
-  - security.openshift.io
-  resourceNames:
-  - anyuid
-  resources:
-  - securitycontextconstraints
-  verbs:
-  - use
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: {{ .Values.namespace }}-argocd-redis-ha-anyuid-scc
+  name: argocd-redis-ha-anyuid-scc
+  namespace: {{ .Values.namespace }}
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: {{ .Values.namespace }}-argocd-redis-ha-anyuid-scc
+  name: system:openshift:scc:anyuid
 subjects:
 - kind: ServiceAccount
-  name: {{ .Values.name }}-argocd-redis-ha
```

Comment:
the first `argocd` is coming from ArgoCD CR name which is default to `argocd` in this chart. If anyone would like to choose a different name, it'll be different for them. Therefore it should stay as `{{ .Values.name }}-argocd-redis-ha` 🙌🏻

Reply:
done ;)